src/styles/partials/components/_pageheader.scss

```diff
@@ -0,0 +1,46 @@
+.dg_page-header {
+	//background-image: url(/sebin/r/t/prototype-subpage-landing-header-bkg.jpg);
+	background-repeat: no-repeat;
+	background-size: cover;
+	background: $pale-blue;
+	color: $white;
+	padding: $padding-largest;
+	text-decoration: none;
+	p {
+		color: $white;
+		font-family: 'Crimson Text', serif;
+		font-size: 25px;
+		font-style: italic;
+		padding-top: 25px;
+	}
+
+	.dg_page-header__box {
+		border-width: 10px;
+		display: flex;
+		flex-wrap: wrap;
+	}
+
+	.dg_page-header__PreTitle {
```

Comment:
the casing for the next two classes should be lower case

.dg_page-header__pre-title

.dg_page-header__title

Reply:
done